.jenkins/dsl/jobs.groovy

```diff
@@ -18,6 +18,28 @@ def getDefaultJobParams(String repoName = 'optaplanner') {
             repository: repoName,
             credentials: "${GIT_AUTHOR_CREDENTIALS_ID}",
             token_credentials: "${GIT_AUTHOR_TOKEN_CREDENTIALS_ID}"
+        ],
+        env: [:],
+        pr: [:]
+    ]
+}
+
+Map getMultijobPRConfig() {
+    return [
+        parallel: true,
+        jobs : [
+            [
+                id: 'Optaplanner',
```

Comment:
For the sake of simplicity, how about making these IDs lowercase?

Reply:
sure we can :) gonna update this